openasip/tools/scripts/install_llvm_17.sh

```diff
@@ -40,13 +40,13 @@ function fetch_llvm {
 
     if ! test -d $llvm_co_dir;
     then
-        git clone --single-branch --depth=1 --branch release/17.x https://github.com/cpc/llvmtce.git $llvm_co_dir\
+        git clone --single-branch --depth=1 --branch release/18.x https://github.com/cpc/llvmtce.git $llvm_co_dir\
```

Comment:
Why 18.x? The script name says 17.

Reply:
I branched the LLVM 17 branch here, so this release still uses LLVM 17 but has the LLVM patches that the new compiler needs here. The old and new branches are not compatible with the RISC-V compiler between OpenASIP revisions, so we cannot patch the new features to the old branch. Perhaps we can just keep this in main like it is and port the new LLVM release on top of this new branch (or a newer one) later?